torrt/notifiers/telegram.py

```diff
@@ -32,13 +33,19 @@ def test_configuration(self):
 
     def send_message(self, msg):
         url = '%s%s/sendMessage' % (self.url, self.token)
-        response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
-        if response.ok:
-            json_data = response.json()
-            if json_data['ok']:
-                LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
+        try:
+            response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        except RequestException as e:
+            LOGGER.error('Failed to send Telegram message: %s' % e)
+        else:
+            if response.ok:
+                json_data = response.json()
+                if json_data['ok']:
+                    LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
```

Comment:
``.debug()`` is probably sufficient not to clutter the log.

Reply:
I'm thinking on it too. But INFO level used from initial commit. So we must add this to release notes.

Reply:
Oh, I see. Let's have it as is.